src/app/portlets/dot-edit-page/layout-properties/dot-layout-properties/dot-layout-properties.component.ts

```diff
@@ -0,0 +1,26 @@
+import { MessageService } from './../../../../api/services/messages-service';
+import { FormGroup } from '@angular/forms';
+import { Component, ViewEncapsulation, Input, OnInit } from '@angular/core';
+
+@Component({
+    encapsulation: ViewEncapsulation.None,
```

Comment:
No... let's encapsulate this and instead of style the svg from outside... pass an param for "header, footer, sidebar-left, sidebar-right" and use that to style it.